packages/braid-design-system/src/lib/utils/source.macro.cjs

```diff
@@ -1,7 +1,8 @@
 const { createMacro } = require('babel-plugin-macros');
 const { default: generate } = require('@babel/generator');
```

Comment:
Right now, because these are `devDependencies` the source gets inlined and the compiled code is 35,000 LOC. If we're to have the snippets be loaded from Metropolis we'll likely have to make these `dependencies` so they don't get bundled up. Thoughts?

Reply:
Solved in [`9187af9` (#1223)](https://github.com/seek-oss/braid-design-system/pull/1223/commits/9187af9c0d756720e3ee878348c727ad1b2f52f4)